packages/react-ui-core/src/Counter/Counter.js

```diff
@@ -34,36 +28,43 @@ export default class IncrementDecrementFilter extends Component {
         <div className={theme.componentContainer}>
           <span
             onClick={this.increment}
-            className={`leftUnit ${theme.contentInnards} ${theme.circle}`}
+            className={classNames('leftUnit', theme.contentInnards, theme.shape)}
           >
-            &#x002B;
+            {this.props.leftOperator}
           </span>
           <span
-            className={`${theme.contentInnards} ${theme.count}`}
+            className={classNames(theme.contentInnards, theme.count)}
           >{`${leftUnit} ${count} ${rightUnit}`}</span>
           <span
             onClick={this.decrement}
-            className={`rightUnit ${theme.contentInnards} ${theme.circle}`}
+            className={classNames('rightUnit', theme.contentInnards, theme.shape)}
           >
-            &#x002D;
+            {this.props.rightOperator}
           </span>
         </div>
       </div>
     )
   }
 }
 
-IncrementDecrementFilter.defaultProps = {
+Counter.defaultProps = {
   theme: {},
   count: 1,
   leftUnit: '',
   rightUnit: '',
+  leftOperatorClick: '',
+  rightOperatorClick: '',
```

Comment:
this is a blank string, but its of type function so it doesn't make sense to use a string here